docs/api/namespace/__getitem__.rst

```diff
@@ -43,3 +43,11 @@
     See also
     --------
     - :ref:`f-expressions` -- user guide on using f-expressions.
+
+    Notes
+    -----
+    .. x-version-changed:: 1.0.0
+
+        :ref:`f-expressions` containing a list/tuple of
+        column names/column positions/column types are
+        accepted within the `j` selector.
```

Comment:
Shouldn't this be in the list of types for `item`? Otherwise, a user might not notice this information down the page.

Reply:
fair point. Let me edit that

Reply:
@st-pasha Would have loved to get ``For example, ``f[0, -1]`` will return the first and last columns.``  on another line (semantic break is what I had in mind) but could not pull it off. Any suggestions?